### PR TITLE
Fix the incorect format of math equation in factorization_ops

### PR DESCRIPTION
**Pull request by imsheridan**
This PR is to fix the incorrect rendering of math equation in [tf.contrib.factorization.WALSModel](https://www.tensorflow.org/versions/r1.7/api_docs/python/tf/contrib/factorization/WALSModel).

This PR fixed the incorrect math equation rendering in above api guides according to the
[Math in markdown](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/docs_src/community/documentation.md#math-in-markdown) guideline. That is using $$, \( and \) to surround the math equation.

Take the [math equation](https://www.tensorflow.org/versions/r1.7/api_docs/python/tf/contrib/factorization/WALSModel#class_walsmodel) as an example to show the fixing result:
Before:
image

After:
image

### CODE REVIEW

**Review by rmlarsen — APPROVED** [review]

**Comment by rmlarsen** [comment]
Thanks!

**Comment by rmlarsen** [comment]
@imsheridan  The "F" should be a subscript on the norm (Frobenius norm). Can you fix that?

File: tensorflow/contrib/factorization/python/ops/factorization_ops.py [hunk 1]
```diff
@@ -51,22 +51,22 @@ class WALSModel(object):
   r"""A model for Weighted Alternating Least Squares matrix factorization.
 
   It minimizes the following loss function over U, V:
-   \\(
+  $$
    \|\sqrt W \odot (A - U V^T) \|_F^2 + \lambda (\|U\|_F^2 + \|V\|_F^2)
```
Comment (rmlarsen) [inline tensorflow/contrib/factorization/python/ops/factorization_ops.py]: "F" should be subscript on the norm.

**Review by rmlarsen — CHANGES_REQUESTED** [review]

**Comment by imsheridan** [comment]
@rmlarsen Could you please help point out which "F"?

File: tensorflow/contrib/factorization/python/ops/factorization_ops.py (hunk 1 above)
Comment (imsheridan) [reply tensorflow/contrib/factorization/python/ops/factorization_ops.py]: Got it, I'll fix it soon

**Comment by imsheridan** [comment]
@rmlarsen Fixed the "F" should be a subscript on the norm (Frobenius norm) as below:
image

**Comment by rmlarsen** [comment]
@imsheridan Thanks!

**Comment by rmlarsen** [comment]
@imsheridan would you mind checking the same issue matrix_solve_ls again?

**Comment by rmlarsen** [comment]
@imsheridan please fix linter errors (probably >80 char lines)

**Comment by imsheridan** [comment]
@rmlarsen I fixed the reported lint errors. Yeah sure I'm checking whether the same issue occurs in matrix_solve_ls.

**Comment by imsheridan** [comment]
@rmlarsen Just FYI, I checked [matrix_solve_ls PR](https://github.com/tensorflow/tensorflow/pull/18022) didn't have the same above subscript "F" issue, so don't needs to change :)

**Review by rmlarsen — APPROVED** [review]

**Pull request merged**